neo/rawio/neuralynxrawio.py

```diff
@@ -87,15 +97,18 @@ def _parse_header(self):
 
             _, ext = os.path.splitext(filename)
             ext = ext[1:]  # remove dot
+            ext = ext.lower()  # make lower case for comparisons
             if ext not in self.extensions:
                 continue
 
-            if (os.path.getsize(filename) <= HEADER_SIZE) and (ext in ['ncs']):
+            # Skip Ncs files with only header. Other empty file types
+            # will have an empty dataset constructed later.
+            if (os.path.getsize(filename) <= NlxHeader.HEADER_SIZE) and ext in ['ncs']:
                 self._empty_ncs.append(filename)
                 continue
 
             # All file have more or less the same header structure
-            info = read_txt_header(filename)
+            info = NlxHeader.buildForFile(filename)
```

Comment:
This function name should be lower case and with underscores for compatibility with PEP8 and consistency with other functions in Neo.

Reply:
Those changes incorporated onto this branch.